src/arche/rules/others.py

```diff
@@ -6,43 +6,44 @@
 import pandas as pd
 
 
-def compare_boolean_fields(source_df, target_df):
+def compare_boolean_fields(source_df: pd.DataFrame, target_df: pd.DataFrame):
     source_bool = source_df.select_dtypes(include="bool")
     target_bool = target_df.select_dtypes(include="bool")
 
     result = Result("Boolean Fields")
     if not fields_to_compare(source_bool, target_bool):
-        result.add_info("No fields to compare")
+        result.add_info("SKIPPED")
         return result
 
-    source_relative_fr = get_bool_relative_frequency(source_bool)
-    target_relative_fr = get_bool_relative_frequency(target_bool)
-    relative_diffs = abs(source_relative_fr - target_relative_fr) * 100
+    lefts = []
+    for c in source_bool:
+        lefts.append(source_bool[c].value_counts(normalize=True).add_prefix(f"{c}_"))
+    b_ = pd.concat(lefts)
 
-    err_diffs = relative_diffs[(relative_diffs > 10).all(1)]
+    rights = []
+    for c in target_bool:
+        rights.append(target_bool[c].value_counts(normalize=True).add_prefix(f"{c}_"))
+    s_ = pd.concat(rights)
+    bool_covs = pd.DataFrame({"source": b_, "target": s_}).fillna(0) * 100
+    bool_covs.name = "Coverage for boolean fields"
+    result.stats.append(bool_covs)
+
+    difs = (bool_covs["source"] - bool_covs["target"]).abs()
+
+    err_diffs = difs[difs > 10]
```

Comment:
This threshold could go in a constant.
Better if some kind of configuration.

Reply:
Ok, makes sense https://github.com/scrapinghub/arche/issues/56